deploy/helm/sumologic/README.md

```diff
@@ -432,6 +432,7 @@ The following table lists the configurable parameters of the Sumo Logic chart an
 | `otellogswindows.metrics.enabled`                                                     | Enable OpenTelemetry Logs Collector for Windows Nodes metrics                                                                                                                                                                                                                                              | `true`                                                                                                                                                                               |
 | `otellogswindows.serviceLabels`                                                       | Add custom labels to OpenTelemetry Logs Collector for Windows Nodes Service                                                                                                                                                                                                                                | `{}`                                                                                                                                                                                 |
 | `otellogswindows.additionalDaemonSets`                                                | OpenTelemetry Logs Collector for Windows Nodes Daemonset per node customization options. See [Best Practices](https://help.sumologic.com/docs/send-data/kubernetes/best-practices/#setting-different-resources-on-different-nodes-for-logs-collector).                                                     | `{}`                                                                                                                                                                                 |
+| `metadata.compression`                                                                | Compression for logs, metrics and events                                                                                                                                                                                                                                                                   | `zstd`                                                                                                                                                                               |
```

Comment:
How about putting this at the top-level, just `compression`? Right now, it affects the log collector and the event collector, so not just metadata. But it doesn't affect the metrics collector.

Reply:
yeah, that's a good point, will make this change.

Reply:
Moved `compression` to the top level. Also added the config option to the otlphttp exporter of the metrics collector.